fault-monitor/operational-status-monitor.cpp

```diff
@@ -19,6 +19,46 @@ namespace status
 {
 namespace monitor
 {
+void Monitor::removeCriticalAssociation(const std::string& objectPath)
+{
+    try
+    {
+        PropertyValue getAssociationValue = dBusHandler.getProperty(
+            objectPath.c_str(), "xyz.openbmc_project.Association.Definitions",
+            "Associations");
+
+        auto association = std::get<AssociationsProperty>(getAssociationValue);
+
+        AssociationTuple critAssociation{
+            "health_rollup", "critical",
+            "/xyz/openbmc_project/inventory/system/chassis"};
+
+        auto it =
+            std::find(association.begin(), association.end(), critAssociation);
+
+        if (it != association.end())
+        {
+            association.erase(it);
+
+            PropertyValue setAssociationValue = association;
```

Comment:
Is this conversion actually required? Can be done in place?

Reply:
done